### PR TITLE
LPS-174237 Remove Stream usages for module: analytics-message-sender-impl

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:analytics-cloud

**Comment by catarineaguiar** [comment]
ci:test:analytics-cloud-acceptance

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/analytics/analytics-message-sender-impl/src/main/java/com/liferay/analytics/message/sender/internal/model/listener/GroupModelListener.java [hunk 1]
```diff
@@ -47,11 +46,15 @@ public List<String> getAttributeNames(long companyId) {
 	public long[] getMembershipIds(User user) throws Exception {
 		List<Group> groups = user.getSiteGroups();
 
-		Stream<Group> stream = groups.stream();
+		long[] membershipId = new long[groups.size()];
```
Comment (dantewang) [inline modules/apps/analytics/analytics-message-sender-impl/src/main/java/com/liferay/analytics/message/sender/internal/model/listener/GroupModelListener.java:49]: @julschong Please rename this, thanks.

**Review by dantewang — APPROVED** [review]

**Comment by julschong** [comment]
see #3338

**Pull request closed without merging**